BUFR_TableD_en_15.csv

```diff
@@ -293,6 +293,64 @@ Category,CategoryOfSequences_en,FXY1,Title_en,SubTitle_en,FXY2,ElementName_en,El
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,001086,Observing platform manufacturer's serial number,,,,Operational
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,008021,Time significance,"Set to 25, nominal reporting time",,,Operational
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,301011,"Year, month, day",,,,Operational
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,301013,"Hour, minute, second",,,,Operational
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,301021,Latitude/longitude (high accuracy),,,,Operational
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,011104,"True heading of aircraft, ship or other mobile platform",,,,Operational
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,002169,Anemometer type,"Set to 2, wind observation through ambient noise (WOTAN)",,,Operational
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,011002,Wind speed,,,,Operational
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,011001,Wind direction,,,,Operational
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,002169,Anemometer type,Set to missing / cancel previous value,,,Operational
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,022032,Speed of sea-surface current,,,,Operational
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,022005,Direction of sea-surface current,,,,Operational
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,301011,"Year, month, day",,,,Operational
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,301013,"Hour, minute, second",,,,Operational
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,008021,Time significance,"Set to 2, time averaged",,,Operational
+15,Oceanographic report sequences,315012,(Sequence for reporting observations from a single glider trajectory profile),,004025,Time period or displacement,"Duration of dive, e.g. -10 (minutes)",,,Operational
```

Comment:
"e.g. -10 (minutes)"

Can a time duration be negative?

Reply:
Yes, this is a time displacement.